SECURITY_CONTACTS

```diff
@@ -0,0 +1,11 @@
+# Reporting a security vulnerability
+
+Please do:
```

Comment:
Just a small nitpick, would `Please do not:` make more sense ?

Reply:
I was thinking that, in case in the future we'd like to extend the list with  both permissive and prohibiting statements then, this way we only need to add them to the list and we would not need to a create a separate "Please do" list.

Reply:
Makes sense.